frontend/components/RunCommand.jsx

```diff
@@ -0,0 +1,77 @@
+import { Box } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+import { Command } from './CommandWidgets';
+import LoadChartWidget from './LoadChartWidget';
+
+const uptime_cmd = 'uptime';
+
+const graph_title = 'Load average';
+
+export default function RunCommand(props) {
+  const [state, setstate] = useState({
+    labels: [],
+    data: [[], [], []],
+    output: '',
+  });
+
+  function extractLoadFromString(str = '') {
+    let list = str.split(',').map(ele => {
+      return ele.trim();
+    });
+
+    let one_min = parseFloat(list[2].split(':')[1]);
+    let five_min = parseFloat(list[3]);
+    let fifteen_min = parseFloat(list[4]);
+
+    return {
+      x_value: new Date(),
```

Comment:
Why generate the `Date` here instead of using the one in the output of `uptime`?